01-introduction.Rmd

```diff
@@ -0,0 +1,204 @@
+*   allows analysts to build and deploy interactive apps and web pages that can be used to communicate analysis and insight to decision–makers
+*   gives analysts the tools to develop reproducible analytical pipelines (RAPs) to automate time–consuming and repetitive tasks, allowing them to focus on interpreting the results
+*   is built in a cloud–based ecosystem that is easy to access remotely from all MoJ IT systems and provides analysts with access to powerful computational resources that can be scaled to meet demand -- this allows analysts to quickly and easily work with big data and perform complex analytical tasks
+*   is secure, resilient and has high availability -- this means analysts can depend on it to store their work and data, and to complete time–sensitive and business–critical projects
+
+## Get started
+
+To get started on the Analytical Platform, you should complete the following steps:
+
+1.  [Read our acceptable use policy](#read-our-acceptable-use-policy).
+2.  [Sign up for a GitHub account](#sign-up-for-a-github-account).
+3.  [Verify your email address for GitHub](#verify-your-email-address-for-github).
+4.  [Configure two-factor authentication for GitHub](#configure-two-factor-authentication-for-github).
+5.  [Request an account from the Analytical Platform team](#request-an-account-from-the-analytical-platform-team).
+6.  [Accept your invitation to the MoJ Analytical Services GitHub organisation](#accept-your-invitation-to-the-moj-analytical-services-github-organisation).
+7.  [Configure two-factor authentication for the Analytical Platform](#configure-two-factor-authentication-for-the-analytical-platform).
+
+### Read our acceptable use policy
+
+Use of the Analytical Platform is subject to our [acceptable use policy](#acceptable-use-policy). You should ensure that you have read and understood this before getting started on the Analytical Platform.
+
+### Sign up for a GitHub account
+
+To sign up for a GitHub account, go to GitHub's [join page](https://github.com/join) and follow the instructions.
+
+*   When signing up, we recommend that you use your work email address.
+*   When instructed to choose your subscription, you should select the free plan.
+*   It is good practice to choose a username that does not contain upper-case characters.
+*   You should use a secure password following best [practice guidelines](https://github.com/ministryofjustice/itpolicycontent/blob/master/content/security/framework/password-standard.md).
+*   We recommend that you use a password manager, such as [LastPass](https://www.lastpass.com/).
+
+### Verify your email address for GitHub
+
+Verifying you email address for GitHub ensures that your account is secure and gives you access to all of GitHub's features.
```

Comment:
And it's necessary to log into AP, I think.

Reply:
Implemented in 6ebbf2f.